README.md

```diff
@@ -588,6 +588,12 @@ var signIn = new OktaSignIn(config);
     helpSupportNumber: '(123) 456-7890'
     ```
 
+- **signOutUrl:** url to redirect when a user clicks the sign out link. If not provided, the widget will navigate to primary auth.
```

Comment:
I am also curious if we want to move to making it more open by providing a click function instead of just a link that we hardcode to redirect since that's what we've been doing in the latest links/buttons we've been adding.